nodes/core/io/21-httprequest.js

```diff
@@ -112,28 +113,42 @@ module.exports = function(RED) {
             }
             if (msg.hasOwnProperty('followRedirects')) {
                 opts.followRedirect = msg.followRedirects;
+            }
+            if (!opts.hasOwnProperty('followRedirect') || opts.followRedirect) {
+                opts.followRedirect = function(res) {
+                    if (this.headers.cookie) {
```

Comment:
Is `this` defined at this point? As it's inside the function passed as `followRedirect` we don't know what the calling scope will be. I suspect `this` should be `node` as that will be in-scope.

Reply:
Sorry - ignore that comment. In this instance we want `this` to be the request instance - so this code is doing the right thing already.